src/viewtransaction/viewTransaction.es6

```diff
@@ -343,12 +345,14 @@ function drawXLines(contract, state, chart) {
   function drawEndTime({ is_path_dependent }) {
     const { is_sold_before_expiry } = state.proposal_open_contract;
 
-    if (exit_tick_time && is_sold_before_expiry) {
+    if (is_path_dependent && exit_tick_time && is_sold_before_expiry) {
+      drawXLine({ line_time: exit_tick_time, label: 'end_time', dashStyle: 'Dash' });
+    }
+    if(!is_path_dependent && exit_tick_time && !is_sold_before_expiry) {
```

Comment:
I think these two could be wrapped in exit_tick_time since both need exit_tick_time?

```
if (exit_tick_time) {
 ...
}
```